src/main/java/com/iridium/iridiumskyblock/IridiumSkyblock.java

```diff
@@ -233,6 +235,8 @@ public void run() {
             }
         }
 
+        this.islandCustomAmountBankGUI = new IslandCustomAmountBankGUI();
```

Comment:
I dont like this, why dont we just create the new class when needed like we do everywhere else. I see no reason to store it here